src/components/AddForm.jsx

```diff
@@ -17,11 +17,12 @@ const handleClick2 = () => {
 }
 
   
-    return<>
+    return<div className='addC'>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the `Add New Book` form is taking the whole space in the right to match the book's container. Also, the page components should be centered kindly make sure to fix this so your design reflects the provided design rules

![Screenshot_20230520_155727](https://github.com/OnlyUnknown/book-store/assets/25227022/b86ce126-317d-4aed-aa60-15dc1d5dabf4)